x/migration/module/autocli.go

```diff
@@ -0,0 +1,35 @@
+package migration
```

Comment:
Let's use autocli this time.

Reply:
:100: - autoCLI only applies to the claim step though, so you'll see `skip: true` until #1046.

Reply:
@bryanchriswhite Noted.

Can you #PUC in two places (in this PR):

1. TODO_UPNEXT pointing to #1046 next to `skip: true`
2. Comment next to `MsgUploadState` explaining why we should never switch to autoCLI (for the reason you outlined above)

Reply:
I'm confused. You say "in this PR", but neither of the things you reference (autoCLI & `MsgCreateMorseAccountState`) exist yet.

Reply:
Done!

[`5d473f0` (#1032)](https://github.com/pokt-network/poktroll/pull/1032/commits/5d473f091af157e267df4aaff885db9e06f89349)

Reply:
:pray: